### PR TITLE
Added incredible header

### PR DESCRIPTION
**Pull request by kpovel**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,5 +7,17 @@
 </head>
 <body>
 <h1>Hello world!</h1>
+<header class="header">
+    <div class="header__logo">
+        <img src="./img/logo.png" alt="logo">
+    </div>
+    <div class="header__menu">
+        <ul>
+            <li><a href="#">Home</a></li>
```
Comment (Yuliia-Karas) [inline index.html:16]: no references

Comment (kpovel) [reply index.html:16]: everything was designed that way, this is a test header!

Comment (Yuliia-Karas) [reply index.html:16]: ok

Comment (Petro-Karas) [reply index.html:16]: no image

**Comment by Petro-Karas** [comment]
no image

**Comment by kpovel** [comment]
@Petro-Karas added logo images

**Pull request merged**